src/models/embeddingsList.ts

```diff
@@ -25,9 +25,9 @@ export class EmbeddingsList
   extends Array<Embedding>
   implements OpenAPIEmbeddingsList
 {
-  model: string;
-  data: Array<Embedding>;
-  usage: EmbeddingsListUsage;
+  model?: string;
+  data?: Array<Embedding>;
+  usage?: EmbeddingsListUsage;
```

Comment:
I think these should be optional to align with the actual interface that's being implemented. Thoughts @aulorbe? https://github.com/pinecone-io/pinecone-ts-client/blob/fd96dcba09aa8751b2d59eb8be9be73380c4adac/src/pinecone-generated-ts-fetch/control/models/EmbeddingsList.ts#L34